documentation/modules/rn-2.5.adoc

```diff
@@ -124,40 +130,113 @@ When migrating VMs that are installed with RHEL 9 as guest operating system from
+
+This issue has been resolved in {project-short} 2.5.3. It is advised to update to this version of {project-short} or later.
+
+For more information, see link:https://access.redhat.com/security/cve/CVE-2023-39322[CVE-2023-39322].
+
+
+.CVE-2023-39321: Processing an incomplete post-handshake message for a QUIC connection can cause a panic
+
+A flaw was found in Golang. This flaw means {project-short} versions before {project-short} 2.5.3 are vulnerable to processing an incomplete post-handshake message for a QUIC connection, which causes a panic. link:https://issues.redhat.com/browse/MTV-693[(MTV-693)]
+
+This issue has been resolved in {project-short} 2.5.3. It is advised to update to this version of {project-short} or later.
+
+For more information, see link:https://access.redhat.com/security/cve/CVE-2023-39321[CVE-2023-39321].
+
+
+.CVE-2023-39319: Flaw in html/template package
+
+A flaw was found in the Golang `html/template` package used in {project-short}. This flaw means {project-short} versions before {project-short} 2.5.3 are vulnerable, as the `html/template` package did not properly handle occurrences of `<script`, `<!--`, and `</script` within JavaScript literals in `<script>` contexts. This flaw could cause the template parser to improperly consider script contexts to be terminated early, causing actions to be improperly escaped, which could be leveraged to perform an `XSS` attack. link:https://issues.redhat.com/browse/MTV-693[(MTV-693)]
+
+This issue has been resolved in {project-short} 2.5.3. It is advised to update to this version of {project-short} or later.
+
+For more information, see link:https://access.redhat.com/security/cve/CVE-2023-39319[CVE-2023-39319].
+
+
+.CVE-2023-39318: Flaw in html/template package
+
+A flaw was found in the Golang `html/template` package used in {project-short}. This flaw means {project-short} versions before {project-short} 2.5.3 are vulnerable as the `html/template` package did not properly handle HMTL-like `""` comment tokens, nor hashbang `\#!` comment tokens. This flaw could cause the template parser to improperly interpret the contents of `<script>` contexts, causing actions to be improperly escaped, which could be leveraged to perform an `XSS` attack. link:https://issues.redhat.com/browse/MTV-693[(MTV-693)]
+
+This issue has been resolved in {project-short} 2.5.3. It is advised to update to this version of {project-short} or later.
+
+For more information, see link:https://access.redhat.com/security/cve/CVE-2023-39318[CVE-2023-39318].
+
+
+.Logs archive file downloaded from UI includes logs related to deleted migration plan/VM
```

Comment:
yep, and `.Log archive file includes logs of a deleted migration plan or VM` from the known issues section should be dropped